staging/src/github.com/openshift/oc/pkg/cli/admin/release/mirror.go

```diff
@@ -442,9 +452,57 @@ func (o *MirrorOptions) Run() error {
 	} else {
 		fmt.Fprintf(o.Out, "\nSuccess\nUpdate image:  %s\nMirrored to: %s\n", to, o.To)
 	}
+
+	printInstallInstructions(o.Out, o.From, o.To, repoMap)
 	return nil
 }
 
+// printInstallInstructions provides detail to the user for using the new mirror for installs.
+// The instructions include yaml formatted output which should be added to the install-config.yaml.
+//
+// Exmaple output:
+//
+//  <descriptive installer usage text>
+//  imageContentSources:
+//  - sources:
+//    - mycompany.com/myrepository/repo                 # destination (mirror) listed first
+//    - registry.svc.ci.openshift.org/ocp/release       # source listed second
+//    - quay.io/openshift-release-dev/ocp-v4.0-art-dev  # repos from image-references
+//
+func printInstallInstructions(out io.Writer, from, to string, repos map[string]struct{}) {
+	type installConfigSubsectionImageContentSources struct {
+		Sources []string `json:"sources"`
+	}
+	type installConfigSubsection struct{
+		ImageContentSources []installConfigSubsectionImageContentSources `json:"imageContentSources"`
+	}
+
+	var repoList []string
+
+	// Append destination (mirror) repo first
+	mirrorRef, _ := imagereference.Parse(to)
```

Comment:
should we be handling errors?

Reply:
I had considered it but since we already parsed these refs previously, it would never trigger an error at the end here.  I'll add them for the same of completeness.

Reply:
Updated.